tests/catch/multiproc/hipIpcEventHandle.cc

```diff
@@ -353,6 +357,22 @@ TEST_CASE("Unit_hipIpcEventHandle_ParameterValidation") {
       REQUIRE(false);
     }
   }
+
+  SECTION("Open handle in process that created it") {
+    hipIpcEventHandle_t event_handle;
+    hipEvent_t event1, event2;
+    HIP_CHECK(hipEventCreateWithFlags(&event1, hipEventDisableTiming | hipEventInterprocess));
+    HIP_CHECK(hipIpcGetEventHandle(&event_handle, event1));
+    HIP_CHECK_ERROR(hipIpcOpenEventHandle(&event2, event_handle), hipErrorInvalidContext);
+  }
+
+  SECTION("Event created with no flags") {
+    hipEvent_t event;
+    hipIpcEventHandle_t event_handle;
+
+    HIP_CHECK(hipEventCreate(&event));
+    HIP_CHECK_ERROR(hipIpcGetEventHandle(&event_handle, event), hipErrorInvalidResourceHandle);
+  }
```

Comment:
Destroy created event using hipEventDestroy

Reply:
The suggestion is implemented. Please proceed with the review.